suncal/suncal.py

```diff
@@ -37,24 +37,33 @@ def create_calendar_events(
         # calculate times of sun event for this date and location
         event_parameters = Celestial(
             timezone=timezone, date=date, longitude=longitude, latitude=latitude
-        ).events[event]
-        # create calendar event and append to list, only if astral could calculate valid datetimes
-        if event_parameters['start'] and event_parameters['end']:
-            gcal_event = GoogleCalEvent(
-                start=GoogleCalTime(
-                    dateTime=event_parameters['start'], timeZone=timezone
-                ),
-                end=GoogleCalTime(
-                    dateTime=event_parameters['end'], timeZone=timezone
-                ),
-                summary=event_parameters['gcal_summary'],
-            )
-            calendar_events.append(gcal_event)
+        ).events.get(event, None)
+
+        # create an all-day event for moon phase
+        if event == 'moonphase':
```

Comment:
I think that this function should not be aware of the type of event - it should be able to proceed in the same way, not matter what event we are looking at. I have a plan for refactoring this, but it won't happen in this PR.